src/Controls/src/Core/HandlerImpl/Switch.Impl.cs

```diff
@@ -1,10 +1,19 @@
-﻿using Microsoft.Maui.Graphics;
+﻿using System.Runtime.CompilerServices;
+using Microsoft.Maui.Graphics;
 
 namespace Microsoft.Maui.Controls
 {
 	/// <include file="../../../docs/Microsoft.Maui.Controls/Switch.xml" path="Type[@FullName='Microsoft.Maui.Controls.Switch']/Docs" />
 	public partial class Switch : ISwitch
 	{
+		protected override void OnPropertyChanged([CallerMemberName] string propertyName = null)
+		{
+			base.OnPropertyChanged(propertyName);
+
+			if (propertyName == IsToggledProperty.PropertyName)
+				Handler?.UpdateValue(nameof(ISwitch.IsOn));
```

Comment:
Is this code preferred here or should it join the others in the  `IsToggledProperty`'s `propertyChanged:` delegate?